### PR TITLE
Switch local-dev/git image to alpine

### PR DESCRIPTION
**Pull request by smlx**
# Checklist
- [ ] Affected Issues have been mentioned in the Closing issues section
- [ ] Documentation has been written/updated.
- [ ] Changelog entry has been written

Switch from a third-party image for git/ssh server used in testing, just use Alpine base images.

# Changelog Entry
Improvement - Reduce third-party dependencies on test images.

# Closing issues
n/a

### CODE REVIEW

File: local-dev/git/Dockerfile [hunk 1]
```diff
@@ -1,12 +1,23 @@
-FROM unixtastic/git-ssh-server
+FROM alpine:3.10
```
Comment (tobybellwood) [inline local-dev/git/Dockerfile]: Can we get this to 3.11 - the other images are mostly running on 3.11 now, alternatively, we could extend on amazeeio/commons to remove the need to manually increment?

```suggestion
FROM alpine:3.11
```

Comment (smlx) [reply local-dev/git/Dockerfile]: Good point, will bump the version.

Comment (smlx) [reply local-dev/git/Dockerfile]: version bumped

**Review by tobybellwood — APPROVED** [review]
approved - all other images are now at 3.11 too

**Pull request merged**